### PR TITLE
Doc(core): Input component

### PR DESCRIPTION
**Pull request by monodyle**

### CODE REVIEW

File: core/src/input/input.stories.tsx
Comment (ng-hai) [inline core/src/input/input.stories.tsx:131]: Grammar "advenced"

File: core/src/input/input.tsx
Comment (thien-do) [inline core/src/input/input.tsx:52]: This is weird.. the change from "1" to "3". I think you would not need this since the numbering reset on every comment block

File: core/src/input/input.tsx
Comment (thien-do) [inline core/src/input/input.tsx:60]: Good.. you know to link to my not-written yet page :))

File: core/src/input/input.tsx
Comment (thien-do) [inline core/src/input/input.tsx:100]: Please help me remove this

File: core/src/input/input.tsx
Comment (thien-do) [inline core/src/input/input.tsx:108]: Good

File: core/src/input/input.tsx
Comment (thien-do) [inline core/src/input/input.tsx:114]: "must be check".. you copy, right :))

File: core/src/input/input.stories.tsx
Comment (thien-do) [inline core/src/input/input.stories.tsx:25]: I don't think "list" should be a part of the "Controlled" group. It also works in uncontrolled input. Should move it to the "Attributes" group.

File: core/src/input/input.stories.tsx
Comment (thien-do) [inline core/src/input/input.stories.tsx:33]: Please help me remove this

**Review by thien-do — CHANGES_REQUESTED** [review]
This is actually pretty good, @monodyle . Only requesting change for:
- Minor issues that I commented
- Also please register a free account at Grammarly and let it checks the text you used here. It's pretty simple, actually, just select all and paste them to their app

**Review by ng-hai — APPROVED** [review]

**Review by thien-do — APPROVED** [review]
This is good. THank you. I'll approve this but if possible please don't merge it yet. I'm having a heavy refactor and when it's done, I'll update this PR for you

**Pull request merged**
